src/GraphQL.Tests/Initialization/SchemaInitializationTests.cs

```diff
@@ -494,9 +495,13 @@ public class SchemaWithFieldStreamResolverOnNonRootSubscriptionField : Schema
     public SchemaWithFieldStreamResolverOnNonRootSubscriptionField()
     {
         var type = new ObjectGraphType { Name = "MyQuery" };
-        type.Field<StringGraphType>("str")
-            .ResolveStream(_ => new Subscription.SampleObservable<string>())
-            .Resolve(_ => "abc");
+        type.AddField(new SubscriptionRootFieldType
```

Comment:
@Shane32 I switched to explicitly construct fields here because there are no methods to call `ResolveStream` yet, `Field` method returns `FieldBuilder`, not `SubscriptionRootFieldBuilder`. As you see user is allowed to use `SubscriptionRootFieldType` in wrong places. Good or bad - my goal here is preserve tests behavior to check error messages. We can design top-level APIs in such a way to prevent all invalid usages like that one.

Reply:
I understand.  As such, this is also not a good example of typical code changes required by users.

Reply:
Yes. For me it was good way to make tests green again.